integration-cli/docker_cli_events_test.go

```diff
@@ -779,7 +779,7 @@ func (s *DockerSuite) TestEventsTop(c *check.C) {
 
 // #13753
 func (s *DockerSuite) TestEventsDefaultEmpty(c *check.C) {
-	dockerCmd(c, "run", "-d", "busybox")
+	dockerCmd(c, "run", "-d", "busybox", "top")
```

Comment:
I think with the '-d' its still racey because its possible the "docker events" gets run before the "docker run -d" is finished.